test/distributed/fsdp/test_fsdp_dtensor_state_dict.py

```diff
@@ -46,9 +46,29 @@ def get_input(self):
         return torch.rand(8, 8, device="cuda")
 
 
+class TestDummyModelUneven(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        torch.manual_seed(0)
+        self.net1 = nn.Sequential(nn.Linear(5, 10), nn.ReLU())
+        self.net2 = nn.Sequential(nn.Linear(10, 15), nn.ReLU())
+        self.net3 = nn.Linear(15, 30)
+        self.net4 = nn.Sequential(nn.ReLU(), nn.Linear(30, 5))
```

Comment:
My original test is pretty funny as there are no Relu for net3. I could not think a reason but a typo. We may want to change it in the future to avoid confusion.